setup.cfg

```diff
@@ -28,5 +28,18 @@ max-line-length = 95
 ignore = E116,E241,E251
 exclude = .git,.tox,.venv,tests/*,build/*,sphinx/search/*,sphinx/pycode/pgen2/*,doc/ext/example*.py
 
+[mypy]
+python_version = 2.7
+show_column_numbers = True
+show_error_context = False
+ignore_missing_imports = True
+follow_imports = skip
+cache_dir = /dev/null
+incremental = True
+check_untyped_defs = True
+warn_no_return = True
+warn_redundant_casts = False
```

Comment:
It seems some configurations are added silently. Why are they needed? please let us know.

Reply:
Oops - these were experiments. I'll remove them